ci/scripts/warnings_baseline.json

```diff
@@ -0,0 +1,4079 @@
+[
```

Comment:
do we need baseline file? if it is needed we should add documentation how to manage this file

Reply:
It will be removed in a separate PR. It will be no longer needed when caches are filled on master branch.